src/styles/main.scss

```diff
@@ -1 +1,12 @@
 // put code here
+@import './utils/variables';
+@import './utils/animations';
+@import './blocks/watch';
+
+body {
```

Comment:
It's better to have a separate file for general styling and keep just imports in the main file